features/src/main/feature/camel-features.xml

```diff
@@ -1617,42 +1617,38 @@
+        <feature version="[4,5)">http-client</feature>
+        <feature>jakarta-servlet</feature>
+        <bundle dependency='true'>mvn:org.apache.olingo/odata-commons-api/5.0.0</bundle>
+        <bundle dependency='true'>mvn:org.apache.olingo/odata-commons-core/5.0.0</bundle>
+        <bundle dependency='true'>mvn:org.apache.olingo/odata-client-api/5.0.0</bundle>
+<!--&lt;!&ndash;        &lt;!&ndash;TODO remove wrap protocol when https://issues.apache.org/jira/browse/OLINGO-1206 is resolved&ndash;&gt;&ndash;&gt;-->
+        <bundle dependency='true'>wrap:mvn:org.apache.olingo/odata-server-api/5.0.0$overwrite=merge&amp;Export-Package=org.apache.olingo.server.api;version=5.0.0,org.apache.olingo.server.api.uri;version=5.0.0,org.apache.olingo.server.api.processor;version=5.0.0,org.apache.olingo.server.api.batch;version=5.0.0,org.apache.olingo.server.api.debug;version=5.0.0,org.apache.olingo.server.api.deserializer;version=5.0.0,org.apache.olingo.server.api.deserializer.batch;version=5.0.0,org.apache.olingo.server.api.etag;version=5.0.0,org.apache.olingo.server.api.prefer;version=5.0.0,org.apache.olingo.server.api.serializer;version=5.0.0,org.apache.olingo.server.api.uri.queryoption;version=5.0.0,org.apache.olingo.server.api.uri.queryoption.apply;version=5.0.0,org.apache.olingo.server.api.uri.queryoption.expression;version=5.0.0,org.apache.olingo.server.api.uri.queryoption.search;version=5.0.0,org.apache.olingo.netty.server.api;version=5.0.0,org.apache.olingo.netty.server.core;version=5.0.0</bundle>
+        <bundle dependency='true'>wrap:mvn:org.apache.olingo/odata-server-core/5.0.0$overwrite=merge&amp;Export-Package=org.apache.olingo.server.core;version=5.0.0,org.apache.olingo.server.core.uri.parser;version=5.0.0</bundle>
+        <bundle dependency='true'>wrap:mvn:org.apache.olingo/odata-client-core/5.0.0$overwrite=merge&amp;Export-Package=org.apache.olingo.client.core;version=5.0.0,org.apache.olingo.client.core.domain;version=5.0.0,org.apache.olingo.client.core.http;version=5.0.0,org.apache.olingo.client.core.communication.request.batch;version=5.0.0</bundle>
+        <bundle dependency='true'>mvn:org.codehaus.woodstox/stax2-api/3.1.4</bundle>
+        <bundle dependency='true'>mvn:commons-io/commons-io/2.15.1</bundle>
+        <bundle dependency='true'>mvn:org.apache.commons/commons-lang3/${commons-lang3-version}</bundle>
+        <bundle dependency='true'>mvn:org.apache.httpcomponents/httpasyncclient-osgi/4.1.5</bundle>
+        <bundle dependency='true'>mvn:com.fasterxml.jackson.core/jackson-core/2.16.2</bundle>
+        <bundle dependency='true'>mvn:com.fasterxml.jackson.core/jackson-databind/2.16.2</bundle>
+        <bundle dependency='true'>mvn:com.fasterxml.jackson.core/jackson-annotations/2.16.2</bundle>
+        <bundle dependency='true'>mvn:com.fasterxml.jackson.dataformat/jackson-dataformat-xml/2.16.2</bundle>
+        <bundle dependency='true'>mvn:com.fasterxml.jackson.module/jackson-module-jaxb-annotations/2.16.2</bundle>
```

Comment:
Please use the Jackson feature

Reply:
done, replaced the 
```
 <bundle dependency='true'>mvn:com.fasterxml.jackson.core/jackson-core/2.16.2</bundle>
        <bundle dependency='true'>mvn:com.fasterxml.jackson.core/jackson-databind/2.16.2</bundle>
        <bundle dependency='true'>mvn:com.fasterxml.jackson.core/jackson-annotations/2.16.2</bundle>
```
with the jackson feature, set the 2 latter dependencies to 2.16.1 to comply with the feature's bundles